dascore/io/dasdae/_compat.py

```
@@ -0,0 +1,102 @@
+        # string attr. Decode only this legacy coord metadata so scan/read can
+        # recover units and steps without reviving general legacy attr unpickling.
+        with contextlib.suppress(
+            AttributeError,
+            EOFError,
+            KeyError,
+            pickle.PickleError,
+            TypeError,
+            UnicodeError,
+            ValueError,
+        ):
+            decoded = pickle.loads(coords.encode("latin1"))
```

Ruff (0.15.21)

[error] `pickle` and modules that wrap it can be unsafe when used to deserialize untrusted data, possible security issue

(S301)